### PR TITLE
Add new LockyRansomware detection event

### LINKED ISSUE #268: Add new detection event for Locky Ransomware semi-supervised model
Related issue aicers/review-web#197

### PR DESCRIPTION
**Pull request by syncpark**
Add new detection event: `LockyRansomware`

Close #268

### CODE REVIEW

File: src/event/dns.rs [hunk 1]
```diff
@@ -136,7 +136,7 @@ impl Match for DnsCovertChannel {
     }
 
     fn level(&self) -> NonZeroU8 {
-        MEDIUM
+        HIGH
```
Comment (msk) [inline src/event/dns.rs:139]: This changes `DnsCovertChannel`. Since this PR is to add `LockyRansomware`, shouldn't this change be in a separate PR, with proper justification?

Comment (syncpark) [reply src/event/dns.rs:139]: Do you think the `LockRansomware` event struct should contain multiple protocol events like Conn or Http instead of only Dns?

Currently `LockyRansomware` model in Hog is learned and created only from Dns events. And the `LockyRansomware` detection event uses same event fields with `DnsCovertChannel`.

If `LockyRansomware` model can be created from various protocol events, a new struct is needed to describe the event.

```rust
struct LockyRansomware {
   service: String, // "Dns"
   fields: Vec<u8>, // Dns event struct of Giganto if service is "Dns"
   confidence: f32
}
```
The `fields` can have type `String` if the `fields` contains JSON string instead of serialized struct.

**Review by msk — APPROVED** [review]
